BildCreatorLibrary/Creator.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using BildingsLibrary;
+using System.Collections;
+
+namespace BildCreatorLibrary
+{
+
+    //Фабрика объектов
+    public class Creator
```

Comment:
Если все методы статические, то и класс стоит сделать статическим.
```suggestion
    public static class Creator
```